pkg/apis/serving/k8s_validation.go

```diff
@@ -566,7 +566,8 @@ func validateContainerPorts(ports []corev1.ContainerPort) *apis.FieldError {
 
 	// Don't allow userPort to conflict with knative system reserved ports
 	if reservedPorts.Has(userPort.ContainerPort) {
-		errs = errs.Also(apis.ErrInvalidValue(userPort.ContainerPort, "containerPort"))
+		errs = errs.Also(apis.ErrInvalidValue(userPort.ContainerPort, "containerPort",
+			fmt.Sprintf("%d is a reserved port, please use a different value.", userPort.ContainerPort)))
```

Comment:
bit of a nit, but looking at the other errors we don't generally add a period `.` at the end
```suggestion
			fmt.Sprintf("%d is a reserved port, please use a different value", userPort.ContainerPort)))
```